src/content/codelabs/dart-cheatsheet.md

```diff
@@ -1778,106 +1777,139 @@ class IntegerHolder {
 
 class IntegerSingle extends IntegerHolder {
   final int a;
-  IntegerSingle(this.a); 
+
+  IntegerSingle(this.a);
 }
 
 class IntegerDouble extends IntegerHolder {
   final int a;
   final int b;
-  IntegerDouble(this.a, this.b); 
+
+  IntegerDouble(this.a, this.b);
 }
 
 class IntegerTriple extends IntegerHolder {
   final int a;
   final int b;
   final int c;
-  IntegerTriple(this.a, this.b, this.c); 
-}
 
+  IntegerTriple(this.a, this.b, this.c);
+}
 
-// Tests your solution (Don't edit!):
+// Tests your solution (Don't edit from this point to end of file):
 void main() {
   final errs = <String>[];
 
-  bool _throwed = false;
-  try {
-    IntegerHolder.fromList([]);
-  } on UnimplementedError {
-    print('Test failed. Did you implement the method?');
-    return;
-  } on Error {
-    _throwed = true;
-  } catch (e) {
-    print('Called IntegerSingle.fromList([]) and got an exception of \n type ${e.runtimeType}.');
-    return;
+  if (!testBadNumberOfArgs(errs, 0)) return;
```

Comment:
It's not quite clear to me what this code is doing. Can I have a brief description?